packages/confirm-dialog/src/vaadin-confirm-dialog-mixin.js

```diff
@@ -288,51 +291,46 @@ export const ConfirmDialogMixin = (superClass) =>
         },
       });
       this.addController(this._confirmController);
+    }
 
-      this._overlayElement = this.$.overlay;
+    updated(props) {
```

Comment:
Let's add JSDoc annotation to prevent this from being shown in API docs::

```suggestion
    /** @protected */
    updated(props) {
```